feedback-module/src/components/Module.js

```diff
@@ -125,6 +143,12 @@ function Module({ pagetitle, endpoint, dir }) {
   };
 
   const changeScreen = (text, nextScreen, feedbackID) => {
+    //adds the screen title as the page title
+    ga && pageTitleAsScreen(ga, t(screen.title));
+
+    //triggers a new event called page_change that shares your current page and next page
+    ga && pageChange(ga, t(screen.title), t(SCREENS[nextScreen].title));
```

Comment:
Do we want the translated version or the English version of the title? Should we perhaps ask design about this?

Reply:
Probably just English because the only people looking at this data will be the agency webmasters

Reply:
The changeScreen function gets called whenever a button is clicked, which doesn't always correspond to whether or not there's a new screen (i.e. if there's an error in one of the input fields, changeScreen will still get called even if the module doesn't actually move on to the next screen). We should probably move this to right before the setScreen gets called (I believe I refactored this changeScreen function quite a bit, so the placement would be something we'd probably have to figure out after merging that).

Reply:
Alternatively, I think we could also just add this to the useEffect where screen is a dependency? not 100% sure on this though